stac_fastapi/extensions/stac_fastapi/extensions/core/filter/filter.py

```diff
@@ -26,9 +26,9 @@ class FilterConformanceClasses(str, Enum):
     See https://github.com/radiantearth/stac-api-spec/tree/v1.0.0-rc.1/fragments/filter
     """
 
-    FILTER = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:filter"
+    FILTER = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter"
```

Comment:
I think this is supposed to be two separate ones -- Filter defines that filter is being used, and Features Filter binds it to the OAFeat endpoint

```
Filter: http://www.opengis.net/spec/ogcapi-features-3/1.0/conf/filter
Features Filter: http://www.opengis.net/spec/ogcapi-features-3/1.0/conf/features-filter
```

Reply:
https://github.com/stac-utils/stac-fastapi/pull/417/commits/40b3ae16741a7044bc87a8597232b24185394fb9